chromadb/api/models/Collection.py

```diff
@@ -229,6 +246,10 @@ def get(
         # Remove URIs from the result if they weren't requested
         if "uris" not in include:
             get_results["uris"] = None
+            
+        if filter_include:
+            keep_keys = ['ids'] + [key for key in include if key != 'ids']
+            return subdict(get_results, keep_keys)
```

Comment:
@thorwhalen, do you find this might break existing downstream projects that may depend on attribute `None` checks?

Also it feels awkward that we return something else than `GetResult`, in this case, a subset dictionary.

Reply:
Not sure I understand: "_(...) break existing downstream projects that may depend on attribute None checks?_". 

But about the `GetResult` TypeDict, know (in case you didn't) that since 3.11, one can make some keys optional, using the [NotRequired](https://docs.python.org/3.11/library/typing.html#typing.NotRequired) type. 

[Here's the pep 655 about it](https://peps.python.org/pep-0655/)